sktime/forecasting/naive.py

```diff
@@ -70,42 +76,40 @@ def fit(self, y_train, fh=None, X_train=None):
         self._set_oh(y_train)
         self._set_fh(fh)
 
-        if self.strategy in ("last", "seasonal_last"):
+        if self.strategy == "last":
             if self.window_length is not None:
-                warn("For the `last` and `seasonal_last` strategy, "
-                     "the `window_length_` value will be ignored.")
+                warn("For the `last` strategy, "
+                     "the `window_length` value will be ignored.")
 
-        if self.strategy in ("last", "mean"):
-            if self.sp is not None:
-                warn("For the `last` and `mean` strategy, "
-                     "the `sp` value will be ignored.")
-
-        if self.strategy == "last":
+        if self.strategy == "last" and self.sp == 1:
```

Comment:
we could rewrite the if condition into a nested if condition, that may make it a bit easier to read: 
```python
if self.strategy == "last":
    if self.sp == 1: 
        ...
    else:
        ...
```